src/Paket.Core/PackageResolver.fs

```diff
@@ -189,6 +189,7 @@ type Resolved = {
 type UpdateMode =
     | UpdatePackage of GroupName * PackageName
```

Comment:
As in, escape ``.``s and treat it as a filter? I think it would work. Hope your regression tests are good up to date though! :D